publication.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Publication of an article
+
+  Scenario: Make an article version ready to publish
+    Given version 1 of article 1234 is in English
```

Comment:
'version _x_ of' everywhere is getting a bit wordy...

Reply:
As is the language

Reply:
Could use a mnemonic code like `article 1234v1`